AdyenUI/Styles/AdyenTextFieldStyle.swift

```diff
@@ -0,0 +1,148 @@
+//
+// Copyright (c) 2025 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Adyen
+import UIKit
+
+public struct AdyenTextFieldStyle {
+    
+    /// The title style.
+    package var title = AdyenLabelStyle(
+        font: .preferredFont(forTextStyle: .footnote),
+        color: UIColor.Adyen.componentSecondaryLabel,
+        textAlignment: .natural
+    )
+    
+    /// The text field's style.
+    package var text = AdyenLabelStyle(
+        font: .preferredFont(forTextStyle: .body),
+        color: UIColor.Adyen.componentLabel,
+        textAlignment: .natural
+    )
+    
+    /// The text field's placeholder text style.
+    package var placeholderText: AdyenLabelStyle?
+
+    /// The color of the background.
+    public var backgroundColor: UIColor
+
+    /// The color of the text.
+    public var textColor: UIColor
+
+    /// The color of the text when the element is active.
+    public var activeColor: UIColor
```

Comment:
I see half the properties package, the rest public. Which one should it be?

Reply:
We do not want to expose `text`, `placeholderText` and `title` property to the merchant. We can change this later if needed.

Reply:
We need to get to to samples stage to see how all of this works in combination together to ensure everything is consistent. For example, as a merchant, I wonder how placeholder or title text color (defined in `AdyenLabelStyle`) would match with the `borderColor` that we allow to change. Without passing the accordingly adjusted `AdyenLabelStyle` this could cause color mismatch. 

Which leads to the question - should we expose the `borderColor`, `textColor` and similar elements that can cause misuse and design inconsistency?